examples/with-temporal/package.json

```diff
@@ -5,23 +5,42 @@
     "build": "next build",
     "start": "next start",
     "type-check": "tsc",
-    "build-worker": "tsc --build temporal/src/worker/tsconfig.json",
-    "build-worker.watch": "tsc --build --watch temporal/src/worker/tsconfig.json",
-    "start-worker": "nodemon -w temporal/lib/ -x 'node temporal/lib/worker || (sleep 5; touch temporal/lib/worker/index.js)'"
+    "build:worker": "tsc --build temporal/tsconfig.json",
+    "build:worker.watch": "tsc --build --watch temporal/tsconfig.json",
+    "start:worker": "cross-env TS_NODE_PROJECT=\"temporal/tsconfig.json\" node --loader ts-node/esm temporal/src/worker.ts",
+    "start:worker.watch": "cross-env TS_NODE_PROJECT=\"temporal/tsconfig.json\" nodemon temporal/src/worker.ts"
   },
   "dependencies": {
     "next": "latest",
+    "node-fetch": "^3.0.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "temporalio": "^0.3.0"
+    "temporalio": "^0.17.2"
   },
   "devDependencies": {
     "@babel/core": "^7.15.0",
-    "@tsconfig/node14": "^1.0.0",
+    "@tsconfig/node16": "^1.0.0",
     "@types/node": "^12.12.21",
+    "@types/node-fetch": "^3.0.3",
     "@types/react": "^17.0.2",
     "@types/react-dom": "^17.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.3.0",
+    "@typescript-eslint/parser": "^5.3.0",
+    "cross-env": "^7.0.3",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-deprecation": "^1.2.1",
```

Comment:
Can we also remove eslint as we don't want to include it by default related example guidelines can be seen here https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples